feature-libs/product-configurator/rulebased/components/group-menu/configurator-group-menu.component.ts

```diff
@@ -86,13 +92,13 @@ export class ConfiguratorGroupMenuComponent {
   ) {}
 
   /**
-   * Fired on key board events, checks for 'enter' and delegates to click.
+   * Fired on key board events, checks for 'enter' or 'space' and delegates to click.
    *
    * @param {KeyboardEvent} event - Keyboard event
    * @param {Configurator.Group} group - Entered group
    */
   clickOnEnter(event: KeyboardEvent, group: Configurator.Group): void {
-    if (event.code === 'Enter') {
+    if (event.code === 'Enter' || event.code === 'Space') {
```

Comment:
Here is the same behavior observed as on the product detail page, namely before navigating to the selected group by clicking `Space` key, the focus is scrolling down within the current group and only then navigates to the selected group

Reply:
Fixed